automl/google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -2778,3 +2807,256 @@ def batch_predict(
         )
         self.__log_operation_info("Batch predict", op)
         return op
+
+
+    def plot_confusion_matrix(
+        self, confusion_matrix, classes, normalize=False, title=None, cmap=plt.cm.Blues
+    ):
+
+        """Plots a confusion matrix. Largely taken from the sklearn plot_confusion_matrix function.
+
+        Example:
+            >>> from google.cloud import automl_v1beta1
+            >>>
+            >>> from google.oauth2 import service_account
+            >>>
+            >>> client = automl_v1beta1.TablesClient(
+            ...     credentials=service_account.Credentials.from_service_account_file('~/.gcp/account.json')
+            ...     project='my-project', region='us-central1')
+            ...
+            >>> d = client.plot_confusion_matrix(cm, ['blue','red','green'])
+            >>>
+
+        Args:
+            confusion_matrix ([[float]]):
+                This is a numpy array of the confusion matrix.
+            classes ([string]):
+                A list of the possible classification values.
+            title (Optional[string]):
+                The title of the confusion matrix.
+            cmap (optional[Color]):
```

Comment:
If there is a `Color` type, where is it imported from?

Reply:
It is imported as a part of matplotlib

Reply:
```suggestion
            cmap (Optional[matpotlib.colors.Colormap]):
```